### DISCUSSION TITLE
Japan position data for RIC and ALB

### DISCUSSION THREAD

**Discussion by AreaXimus1** [theOehrly/Fast-F1#568] [Q&A]
For the first lap, the positional data for Ricciardo and Albon puts them in P2 and P3, respectively. Is this just an issue with how the data is collected?

![image](https://github.com/theOehrly/Fast-F1/assets/154688263/d2ab5d2d-fd92-4845-abf4-0ca96f8ca818)

**Comment by theOehrly** [comment]
That's clearly a bug. I've converted this to issue #569
